FirstTask.py

```diff
@@ -0,0 +1,219 @@
+  
+class Bank:
+
+    account_list = []
+
+    def create_account(self, account):
+        self.account_list.append(account)
+    
+    def login(self, account_number, code):
+        user1=self.find_account(account_number)
+        global logout_check
+        if  logout_check == False:
+            if user1[0]:
+                if code == user1[1].pass_code():
+                    logout_check = False
+                    return True, user1[1]
+            return False, "Account Not Found!"
+
+    def transfer_money(self, sender_account_number, receiver_account_number, amount):
+        user1 = self.find_account(sender_account_number)
+        user2 = self.find_account(receiver_account_number)
+        if user1[0]:
+            user2[1].deposit_bill(amount)
+            user1[1].balance -= amount
+            return True
+        else:
+            return False
+    
+    def compare(self,current_account,other_account):
+        user1 = self.find_account(current_account)
+        user2 = self.find_account(other_account)
+        if user1[0] and user2[0]:
+            if user1[1].check_balance() > user2[1].check_balance():
+                return True, 1
+            if user1[1].check_balance() < user2[1].check_balance():
+                return True, -1
+            if user1[1].check_balance() == user2[1].check_balance():
+                return True, 0
+        else:
+            return False, "Account Not Found!"
+    
+    def find_account(self, account_number):
+        for i in range(len(Bank.account_list)):
+            if account_number == Bank.account_list[i].account_number():
+                return True, Bank.account_list[i]
+        return False, "Account not found!"
+
+    def account_details(self, account_number):
+        check = self.find_account(account_number)
+        if check[0]:
+            check[1].account_details()
+        else:
+            return False
+   
+    def logout():
+        global logout_check
+        logout_check = True
+
+def run():
+    account_number_id = 1000
+    obj = []
+    account_list=Bank()
+    while(1):
+        option = int(input("1) Create Account \n2) Login \n3) Find Account \n0) Exit \nChoose:"))
+        if option == 1:
+            sys('clear')
+            account_number_id += 1
+            name = input("Enter Full Name: ")
+            pass_code = int(input("Enter Passcode "))
+            while(1):
+                account_select = int(input("Select Account Type from following:\n1) Current \n2) Saving \n3) Business\n"))
+                if account_select == 1:
+                    account_type = "Current"
+                    break
+                elif account_select == 2:
+                    account_type = "Saving"
+                    break
+                elif account_select == 3:
+                    account_type = "Business"
+                    break
+                else:
+                    print("\nInvalid Account Type\n")

```

Comment:
We can also create a constant list in `Bank` class: 
```
ACCOUNT_TYPES = ['Current', 'Saving', 'Business']
```
We can then print its values by iterating it here and if the user chooses the correct option (within length of list) then use that `account_type`.